WarmUp1/diff21/src/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+Given an int n, return the absolute difference between n and 21, except return double the absolute difference if n is over 21.
+
+
+diff21(19) → 2
+diff21(10) → 11
+diff21(21) → 0
+*/
+
+//public int diff21(int n)
+
+export function diff21(intParameter: number): number {
```

Comment:
Naming is a bit off here, we know its a parameter based on where it is as well as the type by `: number`